api/v1alpha1/ndb_api_helpers.go

```diff
@@ -326,7 +326,7 @@ func (m *MongodbActionArgs) GetActionArguments(dbSpec DatabaseSpec) []ActionArgu
 		},
 		{
 			Name:  "db_user",
-			Value: dbSpec.Instance.DatabaseInstanceName,
+			Value: "admin",
```

Comment:
Is this change a stop-gap solution or do we plan to accept the db_user value as an input from the users?

Reply:
Stopgap solution. This will be tracked for taking in user inputs too.